test-run.py

```diff
@@ -94,6 +94,8 @@ def main_loop_parallel():
                  format(args.test_timeout), schema='tr_text')
     color_stdout("NO_OUTPUT_TIMEOUT:" . ljust(26) + "{}\n" .
                  format(args.no_output_timeout), schema='tr_text')
+    color_stdout("START_TIMEOUT:" . ljust(26) + "{}\n" .
+                 format(args.server_start_timeout), schema='tr_text')
```

Comment:
I think, it would be better to report it as SERVER_START_TIMEOUT, not just START_TIMEOUT.